R/annotdata.R

```diff
@@ -304,36 +304,42 @@ extractSelectedAnnot <- function(data, threshold, threstype)
 #' @param tmp.dat A slice of \code{annot.dat} which matches data.label.text
 #' It is used when \code{annotation$type} is "Text".
 #' @param prepend Logical; when true, the annotation will be added to the
-#  beginning of data.label.text instead of the end.
+#' beginning of data.label.text instead of the end.
+#' @param tspan Whether to use tspan instead of span.
```

Comment:
Can tspan and span be used interchangeably? What is the difference?

Reply:
If you could a bit more info here that would be great

Reply:
Comments added